documentation/general/dash-sonic-hld.md

```diff
@@ -305,14 +305,31 @@ DASH_ACL_GROUP_TABLE:{{group_id}}
     "guid": {{string}}
 ```
 
+```
+DASH_ACL_SRC_TAG_TABLE:{{tag_name}}
+    "addresses": {{list of prefix}}
+```
+
+```
+DASH_ACL_DST_TAG_TABLE:{{tag_name}}
+    "addresses": {{list of prefix}}
+```
+
+```
+tag_name                  = STRING; unique tag name
+addresses                 = list of ip prefixes ',' separated
+```
+
 ```
 DASH_ACL_RULE_TABLE:{{group_id}}:{{rule_num}}
     "priority": {{priority}}
     "action": {{action}}
     "terminating": {{bool}}
     "protocol": {{list of protocols}}
-    "src_addr": {{list of address}}
-    "dst_addr": {{list of address}}
+    "src_tag_map": {{list of tag name}}
```

Comment:
`src_tag` should be sufficient

Reply:
Why don't we need the dst list?

Reply:
oh i meant src_tag and dst_tag instead of src_tag_map and dst_tag_map